polaris-tokens/tests/token-groups/breakpoints.test.ts

```diff
@@ -0,0 +1,19 @@
+import {
+  breakpoints,
+  breakpointsAlias,
+  BreakpointsAlias,
+} from '../../src/token-groups/breakpoints';
+
+describe('BreakpointsAlias', () => {
+  it('extracts the breakpoint alias from the breakpoints token', () => {
+    function isValidBreakpointsTokenAlias(
```

Comment:
Light suggestion to port the `isKeyOf` and `isValueOf` [type guards](https://github.com/Shopify/polaris/blob/ea2a45bbb9d706fc44100366c7e389f2d25abcf4/polaris-migrator/src/utilities/type-guards.ts) to Polaris tokens. Every test could be updated as follows:
```js
for (const alias of breakpointAlias) {
  expect(isKeyOf(breakpoints, `breakpoints-${alias}`)).toBe(true);
}
```